packages/react-native-oursky/src/RequirePermission.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+import * as React from "react";
+import { Platform } from "react-native";
+import Permissions from "react-native-permissions";
+import OpenSettings from "react-native-open-settings";
+
+import Dialog from "./Dialog";
+import type { LayoutProps as DialogLayoutProps } from "./Dialog";
+
+const AndroidPermission = ["photo"];
+const IOSPermission = ["notification", "photo"];
```

Comment:
Any special reason we only support above permissions? This component should be a generic component that can handle most of the permission cases, at least for common permission like, photo, camera, location and contact.